src/Type/Definition/FieldDefinition.php

```diff
@@ -14,6 +15,7 @@
  * @phpstan-import-type FieldResolver from Executor
  * @phpstan-import-type ArgumentListConfig from Argument
  *
+ * @phpstan-type ArgsMapper callable(array<string, mixed>, FieldDefinition, FieldNode): mixed
```

Comment:
I believe this is the source of all PHPStan errors, since it causes a circular dependency which PHPStan seems to be unable to resolve. Thus, all code that uses a type import from `FieldDefinition` also fails to analyse.

Reply:
See https://github.com/phpstan/phpstan/issues/4637

Reply:
![image](https://github.com/user-attachments/assets/26256cb2-7f42-402a-8e1f-a3bacf83afe8)

Reply:
Maybe we will just have to use a type `mixed` somewhere and kind of handwave the issue.

Reply:
I already tried: @phpstan-type ArgsMapper callable(array<string, mixed>, mixed, mixed): mixed

but same.

  56     Circular definition detected in type alias FieldDefinitionConfig.                                                                                        
  56     Circular definition detected in type alias UnnamedFieldDefinitionConfig.    
  
  Very weird. Will research.

Reply:
Moved it to executor. Maybe its better anyway - similar to ResolveFn.